src/components/application_manager/include/application_manager/resumption/resumption_data_json.h

```diff
@@ -176,7 +182,7 @@ class ResumptionDataJson : public ResumptionData {
   void Persist() OVERRIDE;
 
   resumption::LastState& last_state() const {
-    return last_state_;
+    return last_state_wrapper_->get_accessor().GetMutableData();
```

Comment:
@LitvinenkoIra Don't we need lock this part?

Reply:
@ZhdanovP this function is redundant, so I removed it